services/src/main/java/org/keycloak/authentication/authenticators/client/JWTClientAuthenticator.java

```diff
@@ -267,4 +269,11 @@ public String getId() {
             return Collections.emptySet();
         }
     }
+
+    private List<String> getExpectedAudiences(ClientAuthenticationFlowContext context, RealmModel realm) {
+        String issuerUrl = Urls.realmIssuer(context.getUriInfo().getBaseUri(), realm.getName());
+        String tokenUrl = OIDCLoginProtocolService.tokenUrl(context.getUriInfo().getBaseUriBuilder()).build(realm.getName()).toString();
+        String parEndpointUrl = ParEndpoint.parUrl(context.getUriInfo().getBaseUriBuilder()).build(realm.getName()).toString();
```

Comment:
IMO, PAR is preview feature so that it might be better to include `parEndpointUrl` if PAR feature is activated.

Reply:
+1 . This change is easy, but it will be a bit more complicated to update the test (maybe will be better to move it to ParTest, which rely on PAR feature to be enabled)

Reply:
Yeah .. I was not sure because we kind of need PAR supported. I can do this change.

Reply:
@pedroigor Added comment here based on the team discussion https://github.com/keycloak/keycloak/pull/8266#pullrequestreview-709783674 and approved the PR.